src/components/TypeLayout/TypeLayout.js

```diff
@@ -19,19 +19,10 @@ import {
 
 export const namespace = getComponentNamespace('type-layout__container');
 
-const TypeLayoutCell = ({ children, className, ...other }) => (
-  <StructuredListCell
-    className={classnames(`${namespace}__cell`, className)}
-    {...other}
-  >
-    {children}
-  </StructuredListCell>
-);
-
 // TODO: `2.x` - Remove deprecated prop `bordered`.
 
 /**
- * Type layout component.
+ * Type layouts provide a means to orderly layout terms and definitions.
```

Comment:
I think this is a typo here? 

Perhaps:

```suggestion
 * Type layouts provide an orderly layout of terms and definitions.
```

I think it's just the "orderly" part that seems like the typo but feel free to go with whatever you prefer! 👍